.github/workflows/ci.yml

```diff
@@ -202,6 +203,13 @@ jobs:
           wget https://raw.githubusercontent.com/linux-training-group-1/asset-registry/main/scripts/inserts.sql
           mysql -h localhost -P 3306 --protocol=tcp -u root --password=password < table.sql || true
           mysql -h localhost -P 3306 --protocol=tcp -u root --password=password < inserts.sql || true
+      - name: Run E2E Test
+      # TODO: Need to run the E2E test on Staging Environment
+        run: |
+          wget https://raw.githubusercontent.com/linux-training-group-1/asset-registry/main/tests/test_app.py
```

Comment:
I think integration tests should run on uint+integration testing phase, AND
Selenium tests should run on the staging env.

Reply:
Yeah, in this PR the selenium tests are in the staging environment. But I couldn't test those because the workflow is skipping the staging part